CPS-XXXX/README.md

```diff
@@ -0,0 +1,134 @@
+---
+CPS: XXXX
+Title: Node Behavior during Low Participation
+Status: Draft
```

Comment:
```suggestion
Status: Open
```
@nfrisby please note marking `Draft` review status on GitHub is enough to indicate this is a draft; while the CPS itself will be `Open` until considered solved.  There is no `Draft` status for either CIPs or CPSs.

Likewise the CIP header of https://github.com/cardano-foundation/CIPs/pull/974 can be changed to `Proposed` since that is most likely the way it would be reviewed (once emerging from `Draft` review status) and merged.

Reply:
I hear you. I try to be as explicit as possible, so I like that the rendered documents says Draft for now.

Reply:
sure, although other editors & reviewers are likely to call attention to the same item (though hopefully they will see your stipulation) since it would be a big headache if the document were inadvertently merged this way (i.e. web parsers like `cips.cardano.org` not working right & needing a further PR to clean it up).  As far as I'm concerned it's in your hands. 🙏